requirements.txt

```diff
@@ -1,10 +1,7 @@
 ##eth-brownie==1.19.3
-git+https://github.com/BalancerMaxis/bal_addresses@0.8.2
-git+https://github.com/Tritium-VLK/brownie@master
-prettytable==3.7.0
-dotmap==1.3.30
-pathlib==1.0.1
-pandas==1.5.3
+git+https://github.com/BalancerMaxis/bal_addresses@0.9.4
+git+https://github.com/BalancerMaxis/brownie@v1.20.x
```

Comment:
it doesnt look like you can use `.x` in the tag, and expect it to understand that that is a wildcard. needs to be an exisiting tag that it can match to.

> pipenv.patched.pip._internal.exceptions.InstallationError: Could not detect requirement name for 'git+https://github.com/BalancerMaxis/brownie@v1.20.x', please specify one with #egg=your_package_name

Reply:
`v1.20.x` is a branch name